internal/k8stest/k8s_data_helpers.go

```diff
@@ -20,13 +21,19 @@ func HostEndpoint(t *testing.T) string {
 		return "host.docker.internal"
 	}
 
+	host := os.Getenv("HOST_ENDPOINT")
```

Comment:
If that's only set in the workflow, how developers will know that they need to set this? Maybe that needs to be documented somehow.